src/lib/utilities/decode-payload.test.ts

```diff
@@ -74,71 +70,17 @@ describe(decodePayload, () => {
 
 describe(convertPayloadToJson, () => {
   afterEach(() => {
-    resetLastDataConverterSuccess();
+    resetLastDataEncoderSuccess();
   });
-  it('Should convert a payload through data-converter and set the success status when the websocket is set and the websocket connects', async () => {
-    const ws = new WS('ws://localhost:1337');
 
-    // We need to respond to the websocket messages with the requestID so the
-    // websocket as promised library can resolve the promises correctly without
-    // the requestId it won't properly resolve
-    ws.nextMessage.then((data) => {
-      try {
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const dataz = JSON.parse(data as any);
-
-        ws.send(
-          JSON.stringify({
-            requestId: dataz.requestId,
-            content: { Transformer: 'OptimusPrime' },
-          }),
-        );
-      } catch (e) {
-        // ignore errors, test should fail if we don't respond
-      }
-    });
-
-    const websocket = createWebsocket('1337');
-    await ws.connected;
-
-    const convertedPayload = await convertPayloadToJson(
-      JSON.parse(JSON.stringify(workflowStartedEvent)),
-      websocket,
-    );
-    convertPayloadToJson(JSON.parse(JSON.stringify(workflowStartedEvent)), {});
-    expect(convertedPayload).toEqual(dataConvertedWorkflowStartedEvent);
-
-    const dataConverterStatus = get(lastDataConverterStatus);
-    expect(dataConverterStatus).toEqual('success');
-
-    ws.close();
-    WS.clean();
-  });
-
-  it('Should fail converting a payload through data-converter and set the status to error when the websocket is set and the websocket fails connection', async () => {
-    const websocket = createWebsocket('break', {
-      timeout: 1,
-    });
-
-    const convertedPayload = await convertPayloadToJson(
-      JSON.parse(JSON.stringify(workflowStartedEvent)),
-      websocket,
-    );
-
-    expect(convertedPayload).toEqual(dataConvertedFailureWorkflowStartedEvent);
-
-    const dataConverterStatus = get(lastDataConverterStatus);
-    expect(dataConverterStatus).toEqual('error');
-  });
-
-  it('Should skip converting a payload and set the status to notRequested when the websocket and port is not set', async () => {
+  it('Should skip converting a payload and set the status to notRequested when the encoder endpoint is not set', async () => {
     const convertedPayload = await convertPayloadToJson(
```

Comment:
Hmm we definitely want to keep these tests around to validate the various feedback states for the data encoder.

Reply:
I'll try and refactor to fit the flow without websockets.